### ISSUE TITLE
How to find IK solution without rotation (robot with 3DoFs)?

### ISSUE THREAD

**Issue by AlanSunHR** [NVlabs/curobo#538]
1. cuRobo installation mode (choose from [python, isaac sim, docker python, docker isaac sim]):
python
1. python version:
3.10.18
1. Isaac Sim version (if using): 
NA
------
I want to use the IK solver to find solutions for a 3DoF robot. As it is not possible for a 3DoF robot to specify a 6D goal, I try to provide only goal positions, and I set the rotation_threshold to a very large value (1000). But the ik_solver cannot converge to solutions with position error less than 0.01 (while other non-parallel solvers can).

Do I miss any configuration, or is it currently not possible to ignore the rotation?

**Comment by AlanSunHR** [comment]
From https://github.com/NVlabs/curobo/issues/493 it seems that I should specify a PoseCostMetric with an ignored rotation metric.

So what I need to do is to add the following code:
```python
from curobo.rollout.cost.pose_cost import PoseCostMetric
post_cost_metric = PoseCostMetric(
    reach_partial_pose=True, 
    reach_vec_weight=torch.tensor([0,0,0,1,1,1])
)
ik_solver.update_pose_cost_metric(post_cost_metric)
